.github/workflows/chromatic-push.yml

```diff
@@ -25,4 +24,7 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-        exitOnceUploaded: true # We don't want it to fail CI, we'll be using the GitHub Check
\ No newline at end of file
+        exitOnceUploaded: true # We don't want it to fail CI, we'll be using the GitHub Check
+        # Assume that these changes are acceptable, this will help us maintain the baselines for stories that change each build
+        # We should remove this once our builds are consistent
```

Comment:
If we will auto accept all changes what chromatic will be for?

Reply:
It's in master only.

UI test tracks baselines.
PR review tracks branches.

By auto accepting changes the basline on master and the master branch will be the same for new PR's. At the moment sometimes you get a UI change on master, and PR review will show that as a change. But because the baseline hasn't been accepted it won't show in UI review.

We need to work on fixing these issues. Maybe we should just exclude the stories that change... After writing this, we probably don't need this if I've excluded them all. I'll update the PR.